app/controllers/owners/menus_controller.rb

```diff
@@ -14,6 +14,26 @@ def create
     end
   end
 
+  def edit
+    @menu = Menu.find(params[:id])
+    @restaurant = Restaurant.find(params[:restaurant_id])
```

Comment:
You created a restaurant_params method that you aren't using anywhere (like here, where you could use it). I'm not sure it makes any difference.